routing_model/index.html

```diff
@@ -729,14 +729,16 @@ <h2>Process Adult/Elder promotion and demotion needed from last check</h2>
     The content of the NewSectionInfo parsec event that reached consensus.
     </p>
   </div>
-  <button class="collapsible">set_elder_status</button>
+  <button class="collapsible">update_elder_status</button>
   <div class="content">
-    <p>This function updates the chain to reflect the eldership status of a given node.<br>
-    Input:
-    <lu>
-      <li> node: PublicId
-      <li> is_elder: bool
-    </lu>
+    <p>This function updates the eldership status of each node in the chain based on the new section info.<br>
+    It could look something like this:<br>
```

Comment:
I'm not sure it is sensible to have pseudo code here since you then have to document the sub function.

Maybe just:
"<p>This function updates the eldership status of each node in the chain based on the new section info: Only nodes with their public id in new_section_info are elders.<br>"
+ document intput parameter of update_elder_status like you did for other.